docs/tasks/configure-pod-container/configure-projected-volume-storage.md

```diff
@@ -39,7 +39,7 @@ Here is the configuration file for the Pod:
 1. Verify that the Pod's Container is running, and then watch for changes to
 the Pod:
 
-       kubectl get --watch pod test-projected-volume
+       kubectl get pod test-projected-volume -w
```

Comment:
from a documentation perspective, this is far less understandable to me. So while correct, I don't think the terse-ness that the short option provides is valuable in this case

Reply:
Agreed. `--watch` is much more readable than `-w` tacked on to the end. I'm going to close this PR.